lib/OrganizationsFieldArray/OrganizationsFieldArray.js

```diff
@@ -1,144 +1,83 @@
+    static defaultProps = {
+      addOrganizationBtnLabel: <FormattedMessage id="stripes-erm-components.organizations.addOrganization" />,
+      isEmptyMessage: <FormattedMessage id="stripes-erm-components.organizations.noOrganizations" />,
+      organizationPickerComponent: OrganizationPicker,
     }
 
-    return undefined;
-  }
-
-  renderEmpty = () => (
-    <Layout className="padding-bottom-gutter" data-test-org-empty-message>
-      {this.props.isEmptyMessage}
-    </Layout>
-  );
-
-  renderOrganizations = () => {
-    const { items, name, onDeleteField } = this.props;
-
-    const orgs = items.map((org, index) => (
-      <Row
-        data-test-org
-        key={index}
-      >
-        <Col xs={6}>
-          <Field
-            component={this.props.organizationPickerComponent}
-            data-test-org-name
-            id={`${name}-nameOrg-${index}`}
-            name={`${name}[${index}].org`}
-            validate={[required, this.validateDuplicateEntries]}
-          />
-        </Col>
-        <Col xs={5}>
-          <FormattedMessage id="stripes-erm-components.organizations.selectRole">
-            {placeholder => (
-              <Field
-                component={Select}
-                data-test-org-role
-                dataOptions={this.props.roles}
-                id={`${name}-role-${index}`}
-                name={`${name}[${index}].role`}
-                placeholder={placeholder}
-                validate={required}
-              />
-            )}
-          </FormattedMessage>
-        </Col>
-        <Col xs={1}>
-          <IconButton
-            data-test-org-delete-button
-            icon="trash"
-            id={`${name}-delete-${index}`}
-            onClick={() => onDeleteField(index, org)}
-          />
-        </Col>
-      </Row>
-    ));
-
-    return (
-      <React.Fragment>
-        <Row>
-          <Col xs={6}>
-            <Label required tagName="span">
-              <FormattedMessage id="stripes-erm-components.organizations.name" />
-            </Label>
-          </Col>
-          <Col xs={5}>
-            <Label required tagName="span">
-              <FormattedMessage id="stripes-erm-components.organizations.role" />
-            </Label>
-          </Col>
-        </Row>
-        {orgs}
-      </React.Fragment>
+    renderEmpty = () => (
+      <Layout className="padding-bottom-gutter" data-test-org-empty-message>
+        {this.props.isEmptyMessage}
+      </Layout>
     );
-  }
 
-  render() {
-    const { items, onAddField } = this.props;
+    renderOrganizations = (items) => {
+      const { onMarkforDeletion, name, roles, organizationPickerComponent, onDeleteField, onReplaceField } = this.props;
+      const orgs = items.map((organization, index) => {
+        return (
+          <EditOrganizationCard
+            name={name}
+            organization={organization}
+            index={index}
+            items={items}
```

Comment:
Can you please be careful and conscious about the prop passing you're doing? I don't see why `items` needs to be passed in, and it's not declared in the prop types for `EditOrganizationCard`. This kind of stuff has come up a lot of times during this PR and it's tough to constantly scroll back and forth through Github trying to figure whether I'm missing something or not.